sourcegraph/about
# Add vulnerabilities use case page

Closes #5172 by adding a `vulnerabilities` page under a new `use-cases` pages directory. Also moves `pages/use-cases.tsx` to `pages/use-cases/index.tsx`.

## What are the acceptance criteria? 
- Page matches design specs 
- ^^ on mobile
- All meta is correct
- Carousel behavior everywhere is correct (`code-insights`, `cloud-beta`, this page. We use this component on all of these pages, so any changes we make to it need to look okay in all of these places).
- `BackButton` on get started pages looks okay (that was updated and turned into a shared component).
- Adds link to vulnerabilities page from main index page.
![Screen Shot 2022-03-15 at 5 28 04 PM](https://user-images.githubusercontent.com/43454804/158475673-7bfef68e-d4f4-4b49-ae7f-21f70fe4df4a.png)


## To Test
- Check out the branch and/ or deploy preview and navigate to `/use-cases/vulnerabilities`.
- Check the above criteria.

## Notes
- We are using Material Design icons here, since this is the icon library we use across our properties. The designs use [Phosphor](https://phosphoricons.com/), but CPT has determined that we will not be using icons from this library until we are ready to make a site-wide change. Until that point, we will be using a single icon library for the sake of consistency.
- The `BlogListItem` section behavior on mobile (`Related resources` section) matches the behavior of that component on `code-insights` on mobile, which CPT would like to keep consistent between the pages (using `flex-column` and full-width images on mobile).
- The gradient background on the quote carousel section now matches what's in #5179 — again, for consistency.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201682886788502
